src/main/java/de/zalando/aruha/nakadi/config/RepositoriesConfig.java

```diff
@@ -0,0 +1,66 @@
+package de.zalando.aruha.nakadi.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.zalando.aruha.nakadi.repository.EventTypeRepository;
+import de.zalando.aruha.nakadi.repository.db.EventTypeDbRepository;
+import de.zalando.aruha.nakadi.repository.kafka.KafkaFactory;
+import de.zalando.aruha.nakadi.repository.kafka.KafkaLocationManager;
+import de.zalando.aruha.nakadi.repository.kafka.KafkaRepositorySettings;
+import de.zalando.aruha.nakadi.repository.kafka.KafkaTopicRepository;
+import de.zalando.aruha.nakadi.repository.zookeeper.ZooKeeperHolder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Configuration
+@Profile("!test")
+public class RepositoriesConfig {
+
+    @Autowired
+    private Environment environment;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
I didn't know that one can autowire Configuration. Spring is full of magic :)
Will do that, and will remove circular dependency.